src/ripple/protocol/impl/RippleAddress.cpp

```diff
@@ -486,8 +487,23 @@ void RippleAddress::setAccountPublic (RippleAddress const& generator, int seq)
 }
 
 bool RippleAddress::accountPublicVerify (
-    uint256 const& uHash, Blob const& vucSig, ECDSA fullyCanonical) const
+    Blob const& message, Blob const& vucSig, ECDSA fullyCanonical) const
```

Comment:
We probably need to add a `bool strict` parameter to `sign`:
https://github.com/ripple/rippled/blob/develop/src/ripple/protocol/Sign.h#L36

Alternatively have a function `legacy_sign_ecdsa` that signs using secp256k1 and non-strict.